senza/manaus/constants.py

```diff
@@ -0,0 +1,11 @@
+ELB_REGION_HOSTED_ZONE = {"us-east-1": "Z35SXDOTRQ7X7K",
```

Comment:
Why do we have hardcoded hosted zone IDs here? Is there no better way? This looks brittle...